src/Symfony/Component/Templating/Storage/Storage.php

```diff
@@ -35,7 +35,7 @@ public function __construct(string $template)
      */
     public function __toString()
     {
-        return (string) $this->template;
+        return $this->template;
```

Comment:
The `$template` property is set by the constructor with the `string $template` arg.